lib/galaxy/datatypes/metacyto.py

```diff
@@ -0,0 +1,80 @@
+"""
+MetaCyto analysis datatypes.
+"""
+
+import logging
+
+from galaxy.datatypes.tabular import Tabular
+from galaxy.util import nice_size
+
+log = logging.getLogger(__name__)
+
+
+class mStats(Tabular):
+    """Class describing the table of cluster statistics output from MetaCyto"""
+    file_ext = "metacyto_stats.txt"
+
+    def set_peek(self, dataset, is_multi_byte=False):
+        if not dataset.dataset.purged:
+            dataset.peek = "MetaCyto Cluster Stats"
+            dataset.blurb = nice_size(dataset.get_size())
+        else:
+            dataset.peek = 'file does not exist'
+            dataset.blurb = 'file purged from disk'
+
+    def display_peek(self, dataset):
+        try:
+            return dataset.peek
+        except Exception:
+            return "MetaCyto Cluster Stats"
```

Comment:
Isn't it better to just use the `set_peek()`/`display_peek()` methods of the `Tabular` parent class?

Reply:
Sure, I can remove these.